src/osgPlugins/OpenFlight/VertexPaletteManager.cpp

```diff
@@ -79,8 +79,8 @@ VertexPaletteManager::add( const osg::Geometry& geom )
     if (t && !t2)
         return;
 
-    const bool cpv =( osg::getBinding(geom.getColorArray()) == osg::Array::BIND_PER_VERTEX );
-    const bool npv =( osg::getBinding(geom.getNormalArray()) == osg::Array::BIND_PER_VERTEX );
+    const bool cpv =( geom.getColorBinding() == osg::Geometry::BIND_PER_VERTEX );
```

Comment:
These two lines are the only lines changed in makOsg 3.4, maybe they should be the only 2 lines we keep changed in 3.6

Reply:
Wait, the const for write() is another change.

Reply:
Will keep those two lines and replace the rest with 3.6 versions.   Will keep write() as const unless something in there is trying to modify member data.   Will address on next build.

Reply:
Will use the above two lines and revert the rest.   The const on write will be removed as well since verticesStr will be non-mutable (per next comment).